docs/source/project_documentation/dev_guide/debugging_aac.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
You also need to register this page with the developer guide index page so that it can be reached within the site navigation of the documentation.